### PR TITLE
Updates for release v2022.2 (Backend)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Updates for release v2022.2.

## Changes

Updates Changes, Makefile.PL and Engine.pm.

This PR will not pass the CI tests until https://github.com/zonemaster/zonemaster-ldns/pull/163 has been merged.

## How to test this PR

Review Changes and inspect Makefile.PL and Engine.pm.

### CODE REVIEW

**Review by marc-vanderwal — DISMISSED** [review]
Looks good to me.

File: Changes [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+ - The use of an empty string in the "LANGUAGE.locale"
+   setting is deprecated and will be made illegal in the
+   v2023.1 release. See "docs/Configuration.md#language-section".
+ - The use of two or more locale tags with the same language code
+   in the configuration is deprecated and will be made illegal in the
+   v2023.1 release. See "docs/Configuration.md#language-section".
+
+ [Breaking changes]
+ - Removes database primary key from API method "get_test_results"
+   (#946, #949).
```
Comment (mattias-p) [inline Changes:20]: #949 was included in last release so it should probably be removed from here.

Also it feels a bit silly to bump the major version just for this change. In future releases we should defer less important breaking changes until we've merged a breaking change that really matters.

File: Changes [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
+ [Features]
+ - Provides human readable messages when the test agent dies (#1058)
+ - Adds testcase results (#1055)
+ - Adds Zonemaster-LDNS version to API method "version_info" (#1050)
+
+ [Fixes]
+ - Updates deprecation information in "docs/API.pm" and
+   "docs/Configuration.md" (#1067)
+ - Corrects the license statement in Backend.pm (#1059)
+ - Fixes recursor init (#1057)
```
Comment (mattias-p) [inline Changes:31]: We should use the changes file to describe the user perspective, not the code change. I suggest updating the text of this item to `Fix get_data_from_parent_zone method.`.

**Review by mattias-p — DISMISSED** [review]
If found a couple of nits.

File: Changes (hunk 1 above)
Comment (matsduf) [reply Changes:20]: I know that #949 was in previous release. I just included it as reference.

File: Changes (hunk 2 above)
Comment (matsduf) [reply Changes:32]: I try to do that. In this case I was too lazy to try to find out what that would be. Fixed now.

**Comment by matsduf** [comment]
@mattias-p and @marc-vanderwal, can you re-review and re-approve?

**Review by marc-vanderwal — APPROVED** [review]
Looks good to me.

**Pull request merged**